CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# How to Contribute
+
+Thank you for your interest in contributing to this project.
+There are many ways to contribute:
+
+* Improve existing notebooks (e.g., fix bugs, clarify explanations, optimize code)
+* Report bugs or errors
+* Improve documentation or readability
+
+## Notebook Guidelines
+
+Most of the code in this repository is in the form of Jupyter Notebooks.
+When working with notebooks, please follow these conventions:
+
+* Avoid committing large binary blobs when possible: Unless you are directly modifying inline graphs, do not commit cells with modified large binary blobs, like matplotlib graphs.
+* Check that the notebook runs online: Try running the notebook on Colab and mybinder before opening a pull request.
+* Consistent Style: Follow PEP8 for Python code whenever possible and keep styling consistent.
+* Reusable Code: Avoid hardcoding paths or data. Use relative paths or environment variables when possible.
+
+## Pull Request Process
+
+1. Fork the repository
+2. Create a new feature or bug fix branch (`git checkout -b your-feature-name`)
+3. Commit your changes with a clear message
+4. Push to your fork (`git push origin your-feature-name`)
+5. Open a Pull Request and provide a detailed description
+
+We use GitHub Actions to check notebooks for linting and formatting, please make sure your notebook passes those checks!
+
+## Reporting Issues
+
+If you find a bug or have a suggestion, please open an issue.
```

Comment:
Given the above comment, I would remove "or have a suggestion".  Also, include a link to the issue page (https://github.com/gw-odw/odw/issues).